client/homebrew/pages/newPage/newPage.jsx

```diff
@@ -135,6 +135,13 @@ const NewPage = createClass({
 
 		console.log('saving new brew');
 
+		// Split out CSS to Style if CSS codefence exists
+		if(brew.text.startsWith('```css') && brew.text.indexOf('```\n\n') > 0) {
+			const index = brew.text.indexOf('```\n\n');
+			brew.style = `${brew.style}${brew.text.slice(7, index - 1)}`;
```

Comment:
Maybe insert a newline `\n` between the existing brew.style and the newly inserted part, so we don't end up with

```
styleFromTab {
 color: red;
}stylePastedIn {
 color: blue;
}
```

Reply:
Added in new commit.